tests/test_opencode_sync.py

```
@@ -937,3 +528,326 @@ def test_check_orphans_clean_against_the_real_script() -> None:
     check_orphans = module["check_orphans"]
 
     assert check_orphans(SCRIPT) == []
+
+
+def _write_agent(agents_dir: Path, stem: str) -> None:
+    agents_dir.mkdir(parents=True, exist_ok=True)
+    (agents_dir / f"{stem}.md").write_text("---\nmodel: sonnet\n---\n\nbody\n")
+
+
+def _fake_opencode_run(failing_stems: set[str]):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_fake_opencode_run`

(ANN202)